logstash-input-http_poller.gemspec

```diff
@@ -23,6 +23,9 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'logstash-mixin-http_client', "~> 7"
   s.add_runtime_dependency 'stud', "~> 0.0.22"
   s.add_runtime_dependency 'rufus-scheduler', "~>3.0.9"
+  s.add_runtime_dependency 'logstash-mixin-ecs_compatibility_support', '~>1.3'
```

Comment:
if this is still waiting for the updated `TargetCheck` than we should update the version here